src/pages/uxp/reference-js/Modules/uxp/Persistent File Storage/errors.md

```diff
@@ -109,3 +109,8 @@ Unsupported format type
 Data and Format mismatch
 
 
+
+<a name="module-storage-errors-notsupported" id="module-storage-errors-notsupported"></a>
+
+## NotSupported
+Not supported error
```

Comment:
This doesn't feel like a great description if I'm a developer looking for details on this error. This might have small impact if the actual `error.message` is clear, but this is not visible from this review.